lang/cs/Org.Apache.REEF.IMRU/OnREEF/Client/REEFIMRUClient.cs

```diff
@@ -82,7 +82,7 @@ internal sealed class REEFIMRUClient : IIMRUClient
             }
             catch (Exception e)
             {
-                Exceptions.Throw(e, "Issues in merging PerMapCOnfigGenerator configurations", Logger);
+                Utilities.Diagnostics.Exceptions.Throw(e, "Issues in merging PerMapCOnfigGenerator configurations", Logger);
```

Comment:
In this (and all the other instances), this change leaves an unused 
```
using Org.Apache.REEF.Utilities.Diagnostics;
```
behind. Please remove that as well.